src/lib.rs

```diff
@@ -213,6 +213,13 @@ impl Event {
     /// This is a wrapper of
     /// [tep_record_print_fields](https://www.trace-cmd.org/Documentation/libtraceevent/libtraceevent-field_print.html).
     pub fn print_fields(&self, rec: &Record) {
+        println!("fields: {:?}", self.get_fields(rec));
+    }
+    /// Get each field name follwed by the record's field value according to the field's type.
```

Comment:
nit: "Get" -> "Gets"
We usually regards the function comment as a sentence whose subject is the function name but omitted.